docs/src/TextField.doc.js

```diff
@@ -199,6 +205,53 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="tagsExample"
+    name="Example: Tags"
+    description={`You can include \`Tag\` elements in the input using the \`tags\` prop.`}
+    defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+  const [tags, setTags] = React.useState(['a@pinterest.com', 'b@pinterest.com']);
+  return (
+    <Box padding={2} color="white">
+      <TextField
+        id="emails"
+        label="Emails"
+        onChange={({value}) => {
+          const chunks = value.split(/[ ,;]+/);
```

Comment:
can we rename to tagUnit or tagInput or tagValue or tagText?

chunk reminds me of https://i5.walmartimages.com/asr/d80579d1-3a30-49f6-8fa4-d71b5bde72a0.6881a73c35b1fe1f768d972510cbd6a8.jpeg and makes me get hungry